taichi/runtime/llvm/runtime.cpp

```diff
@@ -1093,7 +1089,7 @@ void element_listgen_root(LLVMRuntime *runtime,
     elem.element = ch_element;
     elem.loop_bounds[0] = c * ch_element_size;
     elem.loop_bounds[1] = std::min((c + 1) * ch_element_size, ch_num_elements);
-    elem.pcoord = refined_coord;
+    elem.pcoord = element.pcoord;
```

Comment:
Could you add a comment explaining that we don't have to refine coordinates, because root snode's num_bits is always 0?